cni-plugins/sriov-passthrough-cni/deploy/multus-cni.yaml

```diff
@@ -156,8 +156,8 @@ spec:
               mountPath: /usr/src/multus-cni/images/
         - name: kube-sriov-passthrough-cni
           command: ["install-plugin"]
-          args: ["/host/opt/cni/bin", "/host/etc/cni/net.d"]
-          image: 'quay.io/pod_utils/sriov-passthrough-cni:v1.0'
+          args: ["/host/opt/cni/bin"]
+          image: 'quay.io/oshoval/prow_cni_multi:v1.1'
```

Comment:
Once there is an ACK on the PR, we can push it to an official repo and change just this yaml on CI,
in order to take those changes.